Source/DataModels/Scene/Scene.cpp

```diff
@@ -253,16 +257,20 @@ void Scene::DestroyGameObject(GameObject* gameObject)
 void Scene::ConvertModelIntoGameObject(const std::string& model)
 {
 	std::shared_ptr<ResourceModel> resourceModel = App->resources->RequestResource<ResourceModel>(model);
-	//resourceModel->Load();
-
+	std::vector<std::shared_ptr<ResourceAnimation>> animations = resourceModel->GetAnimations();
 	std::string modelName = App->fileSystem->GetFileName(model);
 
 	GameObject* gameObjectModel = CreateGameObject(modelName.c_str(), GetRoot());
+
+	if (animations.size() > 0)
```

Comment:
```suggestion
	if (!animations.empty())
```

maybe it's easier to understand like this

Reply:
It's actually the same 'cause the empty function for vectors calculates if size() > 0. So i'm doing this 'cause it should be faster

Reply:
I made a small research and I found this thread that can be interesting. It's not a big issue but it's good to know. Hope it helps!

https://stackoverflow.com/questions/743197/size-vs-empty-in-vector-why-empty-is-preferred

Reply:
In this case 'cause we are using a vector it should take constant time to to the std::size() calculus and it will take O(1). And 'cause we aren't going to change the container to a list it won't be O(n). But sure i can change it if you thing its more readable no problem!

Reply:
both implementations are good for me, use the one you feel more comfortable with. But at least I think we learnt something new from this

Reply:
if you are going to use empty(), I think this one is missing

Reply:
True!

> both implementations are good for me, use the one you feel more comfortable with. But at least I think we learnt something new from this

Yeah for sure!

Reply:
> if you are going to use empty(), I think this one is missing

changed!